src/Microsoft.IdentityModel.Clients.ActiveDirectory/Internal/Flows/AcquireTokenHandlerBase.cs

```diff
@@ -61,15 +61,22 @@ protected AcquireTokenHandlerBase(RequestData requestData)
                 platformInformation.GetProductName(), AdalIdHelper.GetAdalVersion(),
                 AdalIdHelper.GetAssemblyFileVersion(), AdalIdHelper.GetAssemblyInformationalVersion()));
 
-            CallState.Logger.Information(this.CallState,
-                string.Format(CultureInfo.CurrentCulture,
-                    "=== Token Acquisition started:\n\tAuthority: {0}\n\tResource: {1}\n\tClientId: {2}\n\tCacheType: {3}\n\tAuthentication Target: {4}\n\t",
-                    requestData.Authenticator.Authority, requestData.Resource, requestData.ClientKey.ClientId,
-                    (tokenCache != null)
-                        ? tokenCache.GetType().FullName +
-                          string.Format(CultureInfo.CurrentCulture, " ({0} items)", tokenCache.Count)
-                        : "null",
-                    requestData.SubjectType));
+            if (LoggerCallbackHandler.PiiLoggingEnabled)
```

Comment:
we should not need to check for every log. we should be able to just call the method and if PII Logging is enabled then it gets logged.